cmd/clusterctl/client/cluster/mover.go

```diff
@@ -909,7 +917,11 @@ func (o *objectMover) createTargetObject(nodeToCreate *node, toProxy Proxy, muta
 	}
 
 	for _, mutator := range mutators {
-		mutator(obj)
+		err = mutator(obj)
+		if err != nil {
```

Comment:
nit: to match the others
```suggestion
		if err = mutator(obj); err != nil {
```